src/ui/ViewModels/FromUcasViewModel.cs

```diff
@@ -5,6 +5,9 @@ namespace GovUk.Education.ManageCourses.Ui.ViewModels
 {
     public class FromUcasViewModel
     {
+        public bool AllowPreview { get; set; }
+        public bool AllowPublish { get;  set; }
```

Comment:
These properties don't quite match up with the class name.

Not sure what to suggest.

Minor.

Reply:
Renamed the class :+1: